src/components/organisms/WithInformationCard/WithInformationCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
This file is called `WithInformationCard`, but it seems to be implementing a very specific type of 'venue description' information card. Perhaps it should be named more explicitly for that purpose?

Reply:
Maybe `VenueDetailsPopUp` or similar, to match the naming of other similar components?